README.md

```diff
@@ -1,31 +1,56 @@
 # setup-cli
 
-This is a composite GitHub Action that sets up the Databricks CLI (preview version `>=0.100`) in your GitHub Actions workflow.
+Easily install the [Databricks CLI](https://github.com/databricks/cli) in your environment.
+
+Full documentation about installation can be found at:
+* (AWS) https://docs.databricks.com/en/dev-tools/cli/install.html
+* (Azure) https://learn.microsoft.com/en-us/azure/databricks/dev-tools/cli/install
+* (GCP) https://docs.gcp.databricks.com/en/dev-tools/cli/install.html
 
 ## Usage
 
-In your GitHub Actions workflow, use the following step:
+This repository contains an `install.sh` script that can be invoked to install the Databricks CLI.
+
+To install the latest version, run the following command:
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/main/install.sh | sh
+```
+
+To install a specific version, you can replace `main` with a specific release tag:
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/v0.221.1/install.sh | sh
+```
+
+## GitHub Actions
+
+This repository can be used from GitHub Actions.
+
+To always use the latest version of the Databricks CLI, use the action from the `main` branch:
 
 ```yml
 - uses: databricks/setup-cli@main
 ```
 
-<!--
-### Snapshot build
+To pin the version of the Databricks CLI, use the action from a release tag:
```

Comment:
The product docs don't seem to describe this option; we should probably have that there too.

Optional: I don't think we want to make this a "recommendation" do we? Maybe we should add something about when you would use this?

Reply:
Sorry what i was asking is: should we say "this is recommended as a best practice"? And I don't think we should. Instead, we could (optionally) provide some considerations for when to use pinning.

Reply:
I don't think we need to make statements on what is recommended or not. We lead with always using the latest version. Folks reading this probably know what they're doing and when to pin and when not to pin.

To de-emphasize the need to pin, I rewrote to:
> In case you need to use a specific version of the Databricks CLI

Reply:
> I don't think we need to make statements on what is recommended or not. 

+1